$OpenBSD: patch-gnome-panel_panel-menu-items_c,v 1.3 2011/09/12 07:28:54 jasper Exp $

By default it is not possible to run >1 instance of X(7) under OpenBSD
at securelevel(7) 1, hence it is not possible to switch user.

--- gnome-panel/panel-menu-items.c.orig	Sun May 22 10:43:59 2011
+++ gnome-panel/panel-menu-items.c	Sun May 22 10:44:17 2011
@@ -485,8 +485,10 @@ panel_menu_items_create_switch_user (gboolean use_icon
 		item = gtk_image_menu_item_new ();
 	}
 
+#ifndef __OpenBSD__
 	setup_menu_item_with_icon (item, panel_menu_icon_get_size (),
 				   NULL, NULL, NULL, _("Switch User"));
+#endif
 
 	g_signal_connect (item, "activate",
 			  G_CALLBACK (panel_menu_item_activate_switch_user),
